python/ray/tune/utils/util.py

```diff
@@ -184,6 +184,24 @@ def is_nan_or_inf(value):
     return np.isnan(value) or np.isinf(value)
 
 
+def get_current_node_resource_key() -> str:
+    """Get the Ray resource key for current node.
+    It can be used for actor placement.
+
+    If using Ray Client, this will return the resource key for the node that
+    is running the client server.
```

Comment:
For readability, could you add a quick example output?

`resource key` isn't a well-defined term iirc.

Reply:
Added